packages/react-scripts/config/paths.js

```diff
@@ -38,7 +38,9 @@ var nodePaths = (process.env.NODE_PATH || '')
   .split(process.platform === 'win32' ? ';' : ':')
   .filter(Boolean)
   .filter(folder => !path.isAbsolute(folder))
-  .map(resolveApp);
+  .map(resolveApp)
+  // this allows users to `npm link anyOtherProject`
+  .concat([ resolveApp('node_modules') ]);
```

Comment:
I think it is slightly confusing to do it here, since this block is specifically for `NODE_PATH`.
Can we do this directly in Webpack configs instead?
It's fine to have some duplication in case you're worried about it.